upload_artifacts.sh

```diff
@@ -1,21 +1,33 @@
 #!/bin/bash
 
-# Used in .travis.yml
+# This script is used in .travis.yml
+
+set -e
 
 BUCKET=piksi-buildroot-images
 PIKSI_VERSION=v3
+BUILD_VERSION=$(git rev-parse --short HEAD)
 
-set -e
+if [ "$TRAVIS_PULL_REQUEST" == "true" ]; then
+  FOLDER=pull_requests
+else
+  if [ "$TRAVIS_BRANCH" == "master" ]; then
+    FOLDER=master
+  else
+    FOLDER=misc
+  fi
+fi
```

Comment:
every branch and master would get a folder in the root folder, and then there'd be a pr folder - shouldn't be too bad (might be worth making pr `PR` so that wouldn't conflict with a lower case named pr branch).